src/renderer/binary.ts

```diff
@@ -85,8 +85,8 @@ async function downloadBinary(ver: RunnableVersion): Promise<void> {
     // Ensure the target path is empty
     await fs.emptyDir(extractPath);
 
-    const electronFiles = await unzip(zipPath, extractPath);
-    console.log(`Binary: Unzipped ${version}`, electronFiles);
+    await unzip(zipPath, extractPath);
```

Comment:
orthogonal change: `unzip` returns `Promise<void>` so this would always output

```sh
Binary: Unzipped v1.2.3 undefined
```

to console